### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by gfr10598**
Many reorg and cleanup changes.  Separated into several commits for clarity.
1.  Simple TODO added.
2.  Large scale reorg - group into sandbox, staging and prod sections.  Almost zero actual edits.  Pretty hard to follow reorg because so much moves around.
3.  Small scale reordering, and delete PT and SS individual prod targets.
4.  Final edit, some target merging.  Fairly easy to follow diffs relative to 23c71f5.

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/etl/441)

### CODE REVIEW

**Comment by stephen-soltesz** [comment]
Review status: 0 of 1 files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---

*[.travis.yml, line 127 at r1](https://reviewable.io:443/reviews/m-lab/etl/441#-L5B6sOw7u8h-7yphdSW:-L5B6sOw7u8h-7yphdSX:bsrghqy) ([raw file](https://github.com/m-lab/etl/blob/f209c3a148f88776516198a1b5b4dd19852e3ebc/.travis.yml#L127)):*
> ```YAML
> ## Service: queue-pusher -- AppEngine Standard Environment.
> - provider: script
>   script: $TRAVIS_BUILD_DIR/.personalize_deploy.sh
> ```

Drop personalize deploy -- we never actually want this any more. We can remove the script if we want too.

---

*[.travis.yml, line 142 at r1](https://reviewable.io:443/reviews/m-lab/etl/441#-L5B75RFyXWVUOzloe7j:-L5B75RFyXWVUOzloe7k:buyvbcc) ([raw file](https://github.com/m-lab/etl/blob/f209c3a148f88776516198a1b5b4dd19852e3ebc/.travis.yml#L142)):*
> ```YAML
>     /tmp/mlab-sandbox.json $TRAVIS_BUILD_DIR/cmd/etl_worker app-ndt.yaml
>     && INJECTED_PROJECT=mlab-sandbox
>     $TRAVIS_BUILD_DIR/travis/deploy_app.sh ${INJECTED_PROJECT}
> ```

The fix from https://github.com/m-lab/etl/pull/444 was needed b/c evidently travis is doing something funny and variable expansion does not work here the way it does in bash on the command line.

---

*[.travis.yml, line 191 at r1](https://reviewable.io:443/reviews/m-lab/etl/441#-L5B9_2fz4Pmyx2C5RQr:-L5B9_2fz4Pmyx2C5RQs:btuo9sz) ([raw file](https://github.com/m-lab/etl/blob/f209c3a148f88776516198a1b5b4dd19852e3ebc/.travis.yml#L191)):*
> ```YAML
>     INJECTED_PROJECT=mlab-sandbox INJECTED_BUCKET=archive-mlab-sandbox
>     $TRAVIS_BUILD_DIR/travis/deploy_app.sh mlab-sandbox
>     /tmp/mlab-sandbox.json $TRAVIS_BUILD_DIR/cmd/gardener app.yaml
> ```

app-gardener.yaml? 

---

*[.travis.yml, line 226 at r1](https://reviewable.io:443/reviews/m-lab/etl/441#-L5B9lanxSJTPHmm9Ek-:-L5B9lanxSJTPHmm9Ek0:b-pbuv53) ([raw file](https://github.com/m-lab/etl/blob/f209c3a148f88776516198a1b5b4dd19852e3ebc/.travis.yml#L226)):*
> ```YAML
>     && INJECTED_PROJECT=mlab-staging
>     $TRAVIS_BUILD_DIR/travis/deploy_app.sh ${INJECTED_PROJECT}
>     /tmp/${INJECTED_PROJECT}.json $TRAVIS_BUILD_DIR/cmd/etl_worker app-ndt-batch.yaml
> ```

Same issue here with refs to `${INJECTED_PROJECT}`

---

*[.travis.yml, line 302 at r1](https://reviewable.io:443/reviews/m-lab/etl/441#-L5BAIs3H2qd41-UeAI2:-L5BAIs3H2qd41-UeAI3:bmxvw23) ([raw file](https://github.com/m-lab/etl/blob/f209c3a148f88776516198a1b5b4dd19852e3ebc/.travis.yml#L302)):*
> ```YAML
>     all_branches: true
>     tag: true
>     condition: $TRAVIS_TAG == qp-prod-* || $TRAVIS_BRANCH == prod-*
> ```

Let's make queue_pusher part of `small-prod-*`. I imagine this as "ndt" and "everything else".

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/441)*

**Comment by gfr10598** [comment]
Review status: 0 of 1 files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---

*[.travis.yml, line 127 at r1](https://reviewable.io:443/reviews/m-lab/etl/441#-L5B6sOw7u8h-7yphdSW:-L5BDH4i1DVHB2QyfnmU:b-896fix) ([raw file](https://github.com/m-lab/etl/blob/f209c3a148f88776516198a1b5b4dd19852e3ebc/.travis.yml#L127)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

Drop personalize deploy -- we never actually want this any more. We can remove the script if we want too.
</blockquote></details>

Done.

---

*[.travis.yml, line 142 at r1](https://reviewable.io:443/reviews/m-lab/etl/441#-L5B75RFyXWVUOzloe7j:-L5BDOjWyNYurY6Qv8E1:bp3aow6) ([raw file](https://github.com/m-lab/etl/blob/f209c3a148f88776516198a1b5b4dd19852e3ebc/.travis.yml#L142)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

The fix from https://github.com/m-lab/etl/pull/444 was needed b/c evidently travis is doing something funny and variable expansion does not work here the way it does in bash on the command line.
</blockquote></details>

Reverted.

---

*[.travis.yml, line 191 at r1](https://reviewable.io:443/reviews/m-lab/etl/441#-L5B9_2fz4Pmyx2C5RQr:-L5BDdta0bc3PZ_C-415:bomy0h) ([raw file](https://github.com/m-lab/etl/blob/f209c3a148f88776516198a1b5b4dd19852e3ebc/.travis.yml#L191)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

app-gardener.yaml? 
</blockquote></details>

I'm inclined to leave it as app.yaml unless there are multiple files in the same directory.

---

*[.travis.yml, line 226 at r1](https://reviewable.io:443/reviews/m-lab/etl/441#-L5B9lanxSJTPHmm9Ek-:-L5BEWz_P_YmtDthHhgx:bbfns4i) ([raw file](https://github.com/m-lab/etl/blob/f209c3a148f88776516198a1b5b4dd19852e3ebc/.travis.yml#L226)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

Same issue here with refs to `${INJECTED_PROJECT}`
</blockquote></details>

Removed all ${}.  Seems to be fine to pass into script.

---

*[.travis.yml, line 302 at r1](https://reviewable.io:443/reviews/m-lab/etl/441#-L5BAIs3H2qd41-UeAI2:-L5BEogVQkc4PrIicOEU:b-896fix) ([raw file](https://github.com/m-lab/etl/blob/f209c3a148f88776516198a1b5b4dd19852e3ebc/.travis.yml#L302)):*
<details><summary><i>Previously, stephen-soltesz (Stephen Soltesz) wrote…</i></summary><blockquote>

Let's make queue_pusher part of `small-prod-*`. I imagine this as "ndt" and "everything else".
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/441)*

**Comment by gfr10598** [comment]
PTAL

---

Review status: 0 of 1 files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/441#-:-L5BF20Uf1iB_lliNYna:b-83331m)*

**Comment by stephen-soltesz** [comment]
:lgtm:

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[.travis.yml, line 191 at r1](https://reviewable.io:443/reviews/m-lab/etl/441#-L5B9_2fz4Pmyx2C5RQr:-L5BFIcVF14ymw3mlD3J:b70d01s) ([raw file](https://github.com/m-lab/etl/blob/f209c3a148f88776516198a1b5b4dd19852e3ebc/.travis.yml#L191)):*
<details><summary><i>Previously, gfr10598 (Gregory Russell) wrote…</i></summary><blockquote>

I'm inclined to leave it as app.yaml unless there are multiple files in the same directory.
</blockquote></details>

Oh, ha! I missed that we're working in a new directory now! 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/etl/441#-:-L5BFwNtOUzTPWy17-uO:bnfp4nl)*
